localstripe/resources.py

```diff
@@ -817,6 +817,25 @@ def _api_create(cls, source=None, **data):
 
         return obj
 
+    @classmethod
+    def _api_list_all(cls, url, email=None, limit=None, starting_after=None,
+                      **kwargs):
+        try:
+            if email is not None:
+                assert type(email) is str
+        except AssertionError:
+            raise UserError(400, 'Bad request')
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
```

Comment:
We usually put this check first in the functions (see other places in the code).

Reply:
Ok